Software/Development/SemesterProject/App/Include/Constants.h

```diff
@@ -8,77 +8,77 @@
 
 
 //Processor related parameters
-#define CLOCK_FREQUENCY ((double)150e6)     // 150 MHz clock frequency
-#define CLOCK_PERIOD    ((double)6.667e-9 ) // 1/CLOCK_FREQUENCY clock period
-#define SW_FREQ      10000          //Switching frequency [10kHz].
-#define SW_PERIOD_US   100           //Period of the PWM.
-#define SW_PERIOD_S    ((float)100e-6)   //PWM period in s.
-#define INTERNAL_FREQ   90000000    //Internal frequency of the device [90MHz]. Configured in InitSysCtrl().
+#define CLOCK_FREQUENCY             ((float)150e6)          // 150 MHz clock frequency
+#define CLOCK_PERIOD                ((float)6.667e-9)       // 1/CLOCK_FREQUENCY clock period
+#define SW_FREQ                             10000           //Switching frequency [10kHz].
+#define SW_PERIOD_US                        100             //Period of the PWM.
+#define SW_PERIOD_S                 ((float)100e-6)         //PWM period in s.
+#define INTERNAL_FREQ                       90000000        //Internal frequency of the device [90MHz]. Configured in InitSysCtrl().
 
-#define MinPulseWidth	((double)2e-6) //minimum pulse width in s
-#define MinPulseWidthCNT	((unsigned short)300)        //MinPulseWidth/CLOCK_PERIOD (2us)
-#define Half_MinPulseWidthCNT	((unsigned short)150)	//half MinPulseWidth/CLOCK_PERIOD (1us)	
+#define MinPulseWidth	            ((float)2e-6)           //minimum pulse width in s
+#define MinPulseWidthCNT	        ((unsigned short)300)   //MinPulseWidth/CLOCK_PERIOD (2us)
+#define Half_MinPulseWidthCNT	    ((unsigned short)150)	//half MinPulseWidth/CLOCK_PERIOD (1us)
 
 //Mathematical Constants
-#define PI      		((double)(3.141592653589793)) // pi
-#define TWO_PI      	((double)(6.283185307179586)) // 2 * pi
-#define THREE_PI_DIV_2	((double)(4.712388980384690)) // '3*pi/2'
-#define PI_DIV_2 		((double)(1.570796326794897)) // 'pi/2'
-#define PI_DIV_6		((double)(0.523599)) // 'pi/6'
-#define PI_DIV_3		((double)(1.047198)) // 'pi/3'
-#define TWO_PI_DIV_3	((double)(2.094395)) // '2*pi/3'
-#define FOUR_PI_DIV_3	((double)(4.18879)) // '4*pi/3'
-#define FIVE_PI_DIV_3	((double)(5.235988)) // '5*pi/3'
-
-#define  SQRT_2  ((double)1.41421356237309) // sqrt(2)
-#define  SQRT_3  ((double)1.73205080756887) // sqrt(3)
+#define PI      		((float)(3.141592653589793))        // pi
+#define TWO_PI      	((float)(6.283185307179586))        // 2*pi
+#define THREE_PI_DIV_2	((float)(4.712388980384690))        // '3*pi/2'
+#define PI_DIV_2 		((float)(1.570796326794897))        // 'pi/2'
+#define PI_DIV_6		((float)(0.523599))                 // 'pi/6'
+#define PI_DIV_3		((float)(1.047198))                 // 'pi/3'
+#define TWO_PI_DIV_3	((float)(2.094395))                 // '2*pi/3'
+#define FOUR_PI_DIV_3	((float)(4.18879))                  // '4*pi/3'
+#define FIVE_PI_DIV_3	((float)(5.235988))                 // '5*pi/3'
+
+#define  SQRT_2         ((float)1.41421356237309)           // sqrt(2)
+#define  SQRT_3         ((float)1.73205080756887)           // sqrt(3)
```

Comment:
Well it looks to me that many things are not aligned so maybe its how it looks in git?